lib/src/ffmpeg/video_size.dart

```diff
@@ -1,20 +0,0 @@
-class VideoSize {
```

Comment:
I can't remember if I asked this, but why was this file deleted?

Reply:
I didn't see a use and it was causing issues as mentioned here: #4. The only filter it was used in was the scale filter which already had width and height attributes. <br>
It also didn't really follow the [ffmpeg syntax](https://ffmpeg.org/ffmpeg-utils.html#video-size-syntax) which I assume was the main cause of the issue. <br>
I would be open to adding it back if an appropriate use-case arises.

Reply:
Ok. I'm probably using this class in my own projects, so I'll need to check what I was doing with it.

Reply:
It looks like I do use this in my own projects. Can you bring back this file and then adjust whichever filters are broken by it? A filter can parse whatever format it wants, and it can serialize whatever format it wants, regardless of of the internal representation. You can use `VideoSize` inside the filter, or not, but I do need this class to exist and to be published from the package.

Reply:
Alright, I brought back the file and sorted out the files affected.